utils/UpsertServiceEvent.ts

```diff
@@ -0,0 +1,21 @@
+import * as t from "io-ts";
+
+import { RetrievedService } from "io-functions-commons/dist/src/models/service";
+
+/**
+ * Carries information about created or updated service.
+ *
+ * When oldService is defined, the service has been updated, or it has been
+ * created otherwise.
+ */
+export const UpsertServiceEvent = t.intersection([
+  t.interface({
+    newService: RetrievedService,
+    updatedAt: t.number
```

Comment:
you can use [UTCISODateFromString](https://github.com/teamdigitale/io-ts-commons/blob/master/src/dates.ts#L41) to serialize/deserialize `Date`s